external/fv3fit/fv3fit/pytorch/cyclegan/cyclegan_trainer.py

```diff
@@ -33,10 +33,8 @@ class CycleGANNetworkConfig:
     Configuration for building and training a CycleGAN network.
 
     Attributes:
-        generator_optimizer: configuration for the optimizer used to train the
-            generator
-        discriminator_optimizer: configuration for the optimizer used to train the
-            discriminator
+        optimizer: configuration for the optimizer used to train the
```

Comment:
Now that these are combined, can you add to this docstring describing how the discriminator and generator loss configs can be used to set different learning rates for them?

Reply:
This should be covered well enough by the description for `discriminator_weight` and `generator_weight`. There is no way to set different learning rates, but the optimizer update is proportional to both the learning rate and the loss itself (which the weight modifies).